pkg/virt-launcher/virtwrap/api/converter_test.go

```diff
@@ -64,6 +64,16 @@ var _ = Describe("Converter", func() {
 			Expect(xml).To(Equal(convertedDisk))
 		})
 
+		It("should set disk I/O mode if requested", func() {
```

Comment:
Can you set test_id:4560
and can you add test for thread also , test_id:4567? 
(CNV-4567 - [virtual disk] Attach a disk with io=thread)

Reply:
We discussed this offline, adding here for completeness:
Since this is a unit-test, it doesn't need a test id.